docs/variants/novacustom_nv4x_adl/test-matrix.md

```diff
@@ -91,7 +91,7 @@ subjected from before the release of the new binary.
 | 2.   | [Verified Boot support][VBO]               | VBO           | Without VBO006.001 and VBO007.001    |
 | 3.   | [Measured Boot support][MBO]               | MBO           | All                                  |
 | 4.   | [Secure Boot support][SBO]                 | SBO           | Without SBO006.001, SBO007.001 and SBO008.001 |
-| 5.   | [ME neuter support][MNE]                   | MNE           | Without MNE005.001                   |
+| 5.   | [ME neuter support][MNE]                   | MNE           | Without MNE006.001                   |
```

Comment:
@Psotas, and how about any other test matrix dedicated to NC laptops?

Reply:
Updated test matrix for other NC laptops.